index.js

```diff
@@ -21022,35 +21022,51 @@ function BeeSwarmSimulator(DATA){
 
             document.getElementById('newAmuletStats').innerHTML=amulet.map(x=>x.split(' ')[0]+' '+MATH.doGrammar(x.split(' ')[1].replace('INSTANT_CONVERSION','instantConversion').replace('POLLEN','pollen'))).map(x=>x[0]==='+'?'+'+((Number(x.substr(1,x.indexOf(' ')))*100)|0)+'%'+x.substr(x.indexOf(' '),x.length):x).join('<br>').replaceAll('*','x').replace(' Multiplier','').replaceAll(' Passive','').replaceAll('P ','Passive: ')
 
+            
+            document.getElementById('cancelAmuletChange').onclick=function(){

```

Comment:
To test, will start by not allowing any amulet replacement at all, and ensuring that the game does not break.